### PR TITLE
chore(config): add remap coverage support

### PR DESCRIPTION
**Pull request by gregdeane**
#20 

The mods to webpack.config.js cause source maps to break in the browser. I'll see if I can get it to work with the single config file but we may have to add a specific test config.

Use `open coverage/index.html` to see the coverage report.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -43,6 +44,17 @@ module.exports = {
...
+      },
+      {
+        enforce: 'post',
+        test: /\.js$/,
+        loader: 'istanbul-instrumenter-loader',
+        include: __dirname + "/src",
+        exclude: [
+          /\.(e2e|spec)\.js$/,
+          /node_modules/,
+          /vendor/
```
Comment (dami-gg) [inline webpack.config.js]: we currently have no vendor folder but perhaps we should try to move some code there like you have in the user creation mvp, so good that you wrote it ahead of time

File: package.json [hunk 2]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "description": "Reusable navigation header for software connecting to the Zalando Solution Center",
   "scripts": {
+    "rimraf": "rimraf",
     "start": "webpack-dev-server",
+    "pretest": "npm run rimraf -- coverage",
```
Comment (dami-gg) [inline package.json]: wouldn't this work just as rimraf -- coverage without needing the first script?

**Comment by dami-gg** [comment]
:+1:

File: webpack.config.js (hunk 1 above)
Comment (gregdeane) [inline webpack.config.js]: Nope, that's a copy/paste leftover. Removing...

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**